packages/jest-mock/src/testAssets/symbolsAreCool.js

```diff
@@ -0,0 +1 @@
+module.exports = Symbol.for('Bowties.are.cool');
```

Comment:
The bug happens when jest tries to automock a file, placing it in `__mocks__` will bypass it.